data/darktableconfig.xml.in

```diff
@@ -1935,6 +1935,19 @@
     <shortdescription>reduce resolution of preview image</shortdescription>
     <longdescription>decrease to speed up preview rendering, may hinder accurate masking</longdescription>
   </dtconfig>
+  <dtconfig prefs="processing">
+    <name>plugins/lighttable/export/pixel_interpolator_warp</name>
+    <type>
+      <enum>
+        <option>bilinear</option>
+        <option>bicubic</option>
+        <option>lanczos2</option>
+      </enum>
+    </type>
+    <default>bicubic</default>
+    <shortdescription>warp pixel interpolator</shortdescription>
+    <longdescription>pixel interpolator used in modules for rotation, lens correction, liquify, clipping and final scaling (bilinear, bicubic, lanczos2).</longdescription>
```

Comment:
the description of the pixel_interpolator just below also talk about rotation and lens correction. We probably want to adjust.

Reply:
Any reasons not to have lanczos3 here ? Not suited for liquify and other targeted modules ?

Reply:
I am not absolutely sure here, at least lanczos3 might introduce mentioned artefacts to i think we should stay with bicubic in these cases.

Reply:
Yes, I have no problem having bicubic as default, I'm wondering why lanczos3 is not part of the possible options as the other pref. I would think having the 4 choices in both prefs would make sense? But maybe I'm wrong....

Reply:
I think lanczos3 is not a sensible option. Restricting this here was done to make sure artefacts due to a wrong decision here won't propagate to final output. But i have no strong opinion about strictly disabling it as done now.